docs/source/comparison.rst

```diff
@@ -4,68 +4,42 @@ Comparison with Other Frameworks
-
-.. [1] They are also developing `libgpuarray <http://deeplearning.net/software/libgpuarray/>`_
-.. [2] Stateful RNN is a type of RNN implementation that maintains states in the loops. It should enable us to use the states arbitrarily to update them.
-.. [3] This row shows whether each array API supports unified codes for CPU and GPU.
-.. [4] The array backend of Theano does not have compatible interface with NumPy, though most users write code on Theano variables, which is generic for CPU and GPU.
-.. [5] Depending on the frameworks.
-.. [6] Also available in the `Torch RNN package <https://github.com/Element-Research/rnn>`_
-.. [7] Via `Platoon <https://github.com/mila-udem/platoon/>`_
-.. [8] `Experimental as May 2016 <http://deeplearning.net/software/theano/tutorial/using_multi_gpu.html>`_
+This table compares Chainer with other actively developed deep learning frameworks. Content is current as of May 2017.
+
+.. csv-table::
+   :stub-columns: 2
+   :header: ,,"Chainer","PyTorch","TensorFlow","Theano-based","Caffe1/2","Torch7","MXNet","DyNet","PaddlePaddle","DL4J","CNTK","neon","Knet.jl","Darknet","Thinc"
+   
+   "Basics","Language","Python","Python","Python","Python","Python/C++","LuaJIT","Python/others","Python/C++","Python/C++","Java","BrainScript/ Python/C++","Python","Julia","C","Python"
+   ,"Approach","define-by-run","define-by-run","symbolic autograd","symbolic autograd","static","static/ manual grads","symbolic autograd/ manual grads/ define-by-run [1]_","define-by-run","symbolic autograd","static/ manual grads","static/ symbolic autograd","static/ symbolic autograd [2]_","define-by-run","static","callback-based define-by-run"
+   ,"CPU backend","NumPy","TH","Eigen","NumPy","custom","TH","mshadow","Eigen","custom","ND4J","custom","neon","Julia","custom","NumPy"
+   ,"GPU backend","CuPy","THC","Eigen","libgpuarray","custom","THC","mshadow","Eigen","custom","ND4J","custom","neon","custom","custom","CuPy"
+   ,"Primary sponsor","Preferred Networks","Facebook","Google","MILA","Facebook","Facebook","Amazon/Apache","CMU","Baidu","Skymind","Microsoft","Intel Nervana","Koç University","Joe Redmon","Explosion AI"
+   "NNs","CNNs","full","full","full","full","full","full","full","partial","partial","full","full","full","partial","full","none"
+   ,"RNNs","full","full","full","full","partial","full","full","full","full","partial","full","partial","partial","partial","partial"
+   ,"Reverse-mode autograd","Y","Y","Y","Y",,"torch-autograd","Y","Y","Y",,"Y","ngraph","Y",,"with closures"
+   ,"Forward-mode autograd",,,,"Y",,,,,,,,,,,
+   ,"Higher-order grads",,"Y","Y","Y",,,,,,,,,"Y",,
+   ,"Variable-length loops","native","native","while_loop","scan","RNNs only","native","2017","native","RNNs only","none","dynamic axis","none","native","none","native"
+   ,"Per-batch architectures","native","native","fold",,,"torch-autograd","MinPy","native",,,,,"native",,"native"
```

Comment:
Now I know it's been used since the previous table, but it'd be nice if you would change "Per-batch architectures" to "Different architectures per batch" for clarification.